TramsDataApi/Controllers/EstablishmentsController.cs

```diff
@@ -47,6 +49,22 @@ public ActionResult<EstablishmentResponse> GetByUkprn(string ukprn)
             _logger.LogDebug(JsonSerializer.Serialize<EstablishmentResponse>(establishment));
             return Ok(establishment);
         }
+        [HttpGet]
+        [Route("establishment/regions")]
+        public ActionResult<IEnumerable<int>> GetURNsByRegion([FromQuery] string[] regions)
```

Comment:
Do we have any controller tests? Might be worth adding a couple for this.

Reply:
Added some controller tests